### PR TITLE
43 buttons

### PR DESCRIPTION
**Pull request by IagoLast**
#43 
---

- [x] Implementation
- [x] Design Aproval
- [x] UI Test
- [ ] CR

### CODE REVIEW

**Comment by rubenmoya** [comment]
- The buttons should have smooth a transition between states
- The disabled button should have the `not-allowed` cursor
- The disabled button should have the `pointer-events: none` rule
- In safari the button as a margin, which doesn't have in chrome

**Comment by IagoLast** [comment]
@rubenmoya PR updated 👍

File: packages/styles/src/buttons/buttons.scss [hunk 1]
```diff
@@ -60,3 +60,35 @@
...
+  }
+
+  &:active {
+    background-color: var(--as-button-color-focus, #{$color-ui-03});
+  }
+}
+
+a.as-btn.as-btn--primary {
+  &:hover {
+    background-color: var(--as-button-primary-color-focus, darken($color-primary, 20%));
```
Comment (rubenmoya) [inline packages/styles/src/buttons/buttons.scss]: Do we need to interpolate this too?

File: packages/styles/src/buttons/buttons.scss [hunk 2]
```diff
@@ -60,3 +60,35 @@
     font: $font-subheader;
   }
 }
+
+a.as-btn {
+  text-decoration: none;
+
+  &:hover {
+    background-color: var(--as-button-color-focus, #{$color-ui-02});
```
Comment (rubenmoya) [inline packages/styles/src/buttons/buttons.scss]: This is something I want to talk later in the retrospective meeting, should we define these variables on top of the component so we know these variables are created here?

**Comment by piensaenpixel** [comment]
Hi, some things

- Focus state, we should remove the outline
image

- Border should be '0' instead of 1px transparent
image

File: packages/styles/src/buttons/buttons.scss (hunk 2 above)
Comment (IagoLast) [reply packages/styles/src/buttons/buttons.scss]: yes, sure

File: packages/styles/src/buttons/buttons.scss (hunk 1 above)
Comment (IagoLast) [reply packages/styles/src/buttons/buttons.scss]: nope

**Comment by IagoLast** [comment]
@piensaenpixel 

> Focus state, we should remove the outline

how do we give focus feedback then?

> Border should be '0' instead of 1px transparent

So secondary button will be bigger because of its border, is the expected behaviour?

**Comment by piensaenpixel** [comment]
- Focus should have the same design that the press state
- we need use the border-size property to avoid the different size.

**Comment by piensaenpixel** [comment]
BTW now the button is 42px instead 40, remember all the sizes should be multiple of 4

**Comment by piensaenpixel** [comment]
-  Here we have a double border
image

-  And i think the markup should be a nav or at least a list

The rest is ok for me

**Pull request merged**